### PR TITLE
feat: update python version to 3.11

### LINKED ISSUE #68: Update runtimes to python 3.11
☝️

### PR DESCRIPTION
**Pull request by vincentsarago**
closes #68 

To Be discussed: should this be considered as a breaking change? @alukach @sharkinsspatial @emileten

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
-  "name": "cdk-pgstac",
```
Comment (emileten) [inline package-lock.json:2]: this still had `cdk-pgstac` instead of `eoapi-cdk` in it ? 😵‍💫

Comment (vincentsarago) [reply package-lock.json:2]: oh it's interesting, I pushed my own package-lock.json while it seems that when doing a release only the version number was changed in the package-lock.json, not the whole file was update 🤷‍♂️

**Comment by emileten** [comment]
FYI I'll be addressing this as part of https://github.com/developmentseed/eoapi-cdk/pull/74 -- I'll close here once the latter is merged.

**Comment by vincentsarago** [comment]
done in #74

**Pull request closed without merging**